prov/efa/src/rdm/efa_rdm_pke_rtr.c

```diff
@@ -66,6 +66,40 @@ ssize_t efa_rdm_pke_init_longcts_rtr(struct efa_rdm_pke *pkt_entry,
 	return 0;
 }
 
+/**
+ * @brief allcoate an RX entry for a incoming RTR packet
+ *
+ * The RX entry will be allocated from endpoint's OP entry
+ * pool
+ * @param[in]	pkt_entry	received RTR packet
+ *
+ * @return
+ * pointer to the newly allocated RX entry.
+ * NULL when OP entry pool has been exhausted.
+ */
+struct efa_rdm_ope *efa_rdm_pke_alloc_rtr_rxe(struct efa_rdm_pke *pkt_entry)
+{
+	struct efa_rdm_ep *ep = pkt_entry->ep;
+	struct efa_rdm_ope *rxe;
+	struct efa_rdm_rtr_hdr *rtr_hdr;
+
+	rxe = efa_rdm_ep_alloc_rxe(ep, pkt_entry->addr, ofi_op_read_rsp);
+	if (OFI_UNLIKELY(!rxe))
+		return NULL;
+
+	rxe->addr = pkt_entry->addr;
+	rxe->bytes_received = 0;
+	rxe->bytes_copied = 0;
```

Comment:
These are already be initialized via `efa_rdm_ep_alloc_rxe()`

Reply:
Thanks, removed